packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -402,20 +409,40 @@ const main = (opts) => {
                 case HELLO_WORLD_TEST_PROJECT:
                     return generateHelloWorld({projectId: 'hello-world'}, opts)
                 case HELLO_WORLD:
-                    return helloWorldPrompts(opts).then((answers) =>
-                        generateHelloWorld(answers, opts)
-                    )
+                    return helloWorldPrompts(opts).then((answers) => {
+                        const projectId = slugifyName(answers.projectName)
+                        if (!OUTPUT_DIR_FLAG_ACTIVE) {
+                            opts.outputDir = p.join(process.cwd(), projectId)
+                        }
+                        generateHelloWorld(projectId, opts)
+                        return opts.outputDir
+                    })
                 case TEST_PROJECT:
                     return runGenerator(testProjectAnswers(), opts)
                 case RETAIL_REACT_APP_DEMO:
-                    return runGenerator(demoProjectAnswers(), opts)
+                    return Promise.resolve()
+                        .then(() => runGenerator(demoProjectAnswers(), opts))
+                        .then((result) => {
+                            console.log(
+                                '\nTo change your ecommerce back end you will need to update your storefront configuration. More information: https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/configuration-options'
+                            )
+                            return result
+                        })
                 case RETAIL_REACT_APP:
                     console.log(
-                        'For details on configuration values, see https://developer.salesforce.com/docs/commerce/commerce-api/guide/commerce-api-configuration-values\n'
+                        'For details on configuration options, see https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/configuration-options\n'
                     )
-                    return retailReactAppPrompts(opts).then((answers) =>
+                    return retailReactAppPrompts(opts).then((answers) => {
+                        if (!OUTPUT_DIR_FLAG_ACTIVE) {
+                            opts.outputDir = p.join(
+                                process.cwd(),
+                                slugifyName(answers.globals.projectId)
```

Comment:
Just a question (not a suggestion): why slugify the project name when we're using it as the output dir? I understand this makes it consistent with what is set in the package.json name but the behavior is a bit different compared to how --outputDir behaves (since this is not slugify-ed)

Reply:
@vcua-mobify The acceptance criteria:

> Rename the question about MR project id to ask for the name of the project, which will be used as a folder name as well as MRT project id in `pacakage.json`.


I understand that we want to use the same value in two systems with different characters restrictions. That means that the value that we want to use has to meet at least the most restrictive criteria of both systems:

1. Operating Systems have different sets of invalid path characters.

e.g. Windows File systems:
https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file?redirectedfrom=MSDN#naming-conventions

> The following reserved characters:
> 
> "<" (less than)
> ">" (greater than)
> : (colon)
> " (double quote)
> / (forward slash)
> \ (backslash)
> | (vertical bar or pipe)
> ? (question mark)
> "*" (asterisk)
> 

2. MRT project id only allows letters, numbers and the dash symbol.

The slug generated from the project name meets the requirements for both systems.